tutorials/primer/.pages

```diff
@@ -0,0 +1,7 @@
+nav:
+  - ...
+  - elements-shards.md
```

Comment:
I would also add `introduction.md` and `index.md` and remove `...` here.

That way if we add other pages later, we will have to think of the proper order instead of having them popup up in front in unexpected order.

Reply:
Fixed.